.github/workflows/ci.yml

```diff
@@ -36,23 +33,26 @@ jobs:
       - name: Env
         run: env
 
+      - name: Dump GitHub context
+        id: github_context_step
+        run: echo $JSON
+        env:
+          JSON: ${{ toJSON(github) }}
+
```

Comment:
Would this potentially expose any sensitive info? Was this mainly for debugging?

Reply:
I think any sensitive info is masked, for example [in this build](https://github.com/guardian/support-reminders/actions/runs/5576846650/jobs/10188728237) expanding the step shows that the token is masked.

This isn’t something I added, but I did have the same doubts about it. Let’s get rid of it – it’s useful for understanding the structure of the context and checking values, but we can always add the step back for debugging a given execution if needed.

Reply:
Done now